crates/libcgroups/src/v2/devices/controller.rs

```diff
@@ -11,6 +11,12 @@ use oci_spec::runtime::LinuxDeviceCgroup;
 use crate::common::{default_allow_devices, default_devices, ControllerOpt};
 use crate::v2::controller::Controller;
 
+#[cfg(test)]
+use bpf::mock_prog::{attach, bump_memlock_rlimit, detach2, load, query};
+
+#[cfg(not(test))]
+use bpf::prog::{attach, bump_memlock_rlimit, detach2, load, query};
```

Comment:
this can be cleaned up by mockall's `double`, but would require it to be in the release dependencies, which I wanted to avoid